development/tools/cmd/pjtester/README.md

```diff
@@ -0,0 +1,49 @@
+# Prowjob tester
```

Comment:
Please adjust this document to match the [template](https://github.com/kyma-project/community/blob/master/guidelines/repository-guidelines/repository-template/README.md) (adjust the sections titles - **Overview**, **Usage**, **Development** - and structure - if you wish to keep the existing titles, then, if their presence is justified, make them subtitles of the required sections, and add the **Prerequisites** and **Installation** sections).
We have these templates put in place for consistency and simplicity, please stick to them. You can find the templates [here](https://github.com/kyma-project/community/tree/master/guidelines/templates/resources), and if you're not sure which one to use, we also have a [list of the template types and their descriptions](https://github.com/kyma-project/community/blob/master/guidelines/templates/templates-type.md).

Reply:
Done